apps/config-server/Dockerfile

```
@@ -16,6 +16,8 @@ RUN ./gradlew :apps:config-server:clean :apps:config-server:build -x test --no-d
 
 FROM eclipse-temurin:17-jre
 
+RUN apt-get update && apt-get install -y curl && rm -rf /var/lib/apt/lists/*
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)